app/github_repo_fetcher.rb

```diff
@@ -29,8 +29,8 @@ def docs(app_name)
         title = ExternalDoc.title(contents) || filename
         {
           path: "/apps/#{app_name}/#{filename}.html",
-          title: title,
-          markdown: contents,
+          title: title.to_s.force_encoding("UTF-8"),
+          markdown: contents.to_s.force_encoding("UTF-8"),
```

Comment:
Minor: I forgot to suggest this could also have a test to show the kind of thing it's protecting against.

Reply:
The current test setup is [pretty horrific](https://github.com/alphagov/govuk-developer-docs/blob/master/spec/app/github_repo_fetcher_spec.rb#L84-L119) when mocking the actual file contents of a doc. Rather than add to the cruft, I'm hoping to refactor this in a future PR, where I'll retrospectively add a test for UTF-8 conversion 😅

Reply:
Oh yeah, that's not very easy to extend 😬. Sounds like we're on the same page with all the follow-up changes 👍.